demos/list.html

```diff
@@ -1200,6 +1329,32 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        function clickHandler(evt, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          if(evt.target === li) {
+            checkbox.checked = !checkbox.checked;
+          }
+        };
+
+        function addRippleToListItems(list) {
+          var listItems = list.querySelectorAll('.mdc-list-item');
+          var checkboxes = list.querySelectorAll('.mdc-checkbox');
+
+          [].forEach.call(listItems, function(li) {
+            mdc.ripple.MDCRipple.attachTo(li);
+            li.addEventListener('click', function(e) {
+              clickHandler(e, li);
+            });
+          });
+
+          [].forEach.call(checkboxes, function(checkbox) {
+            mdc.ripple.MDCRipple.attachTo(checkbox);
```

Comment:
`mdc.checkbox.MDCCheckbox.attachTo`?

Also, this loop ends up creating a second ripple on `#toggle-rtl-checkbox` (the first is on [lines 1307-1309](https://github.com/material-components/material-components-web/blob/cf2b5a537510f362c9e9eb023e1bd404850a21a6/demos/list.html#L1307-L1309)).

I would remove lines 1307-1309, and replace this loop with:

```js
[].forEach.call(checkboxes, function(checkbox) {
  var cbInstance = mdc.checkbox.MDCCheckbox.attachTo(checkbox);
  if (checkbox.parentElement.classList.contains('mdc-form-field')) {
    var ffInstance = mdc.formField.MDCFormField.attachTo(checkbox.parentElement);
    ffInstance.input = cbInstance;
  }
});
```

Reply:
Will also remove the id `toggle-rtl-checkbox` from the checkbox